modules/repository/init.go

```diff
@@ -71,6 +71,10 @@ func LoadRepoConfig() {
 			}
 
 			for _, f := range customFiles {
+				ext := strings.ToLower(filepath.Ext(f))
+				if ext == ".yaml" || ext == ".yml" {
+					f = f[:len(f)-len(ext)]
+				}
```

Comment:
Are you sure this is right?

You removed the "ext", but you still append the `f` to `files`. Could each item in `files` be read correctly later?